src/promptflow/tests/test_configs/flows/chat_group_eval_history/README.md

```diff
@@ -0,0 +1,43 @@
+# Basic Eval
```

Comment:
this file seems no use?

Reply:
copied from the example repo, could remove it later.